cmd/troubleshoot/builder.go

```diff
@@ -105,14 +105,15 @@ func RunTroubleshootCmd(ctx context.Context, log logd.Logger, namespaceName stri
 		return
 	}
 
-	dynakubes := &dynakube.DynaKubeList{}
+	dks, err := getDynakubes(ctx, log, apiReader, namespaceName, dynakubeFlagValue)
```

Comment:
Small nitpick that was already there before: Why the list before the if and not after